src/stories/components/check-box/check-box.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import CheckBox from '@ses/components/check-box/check-box';
+import { withFigmaComparator } from '@ses/core/utils/storybook/decorators';
+import { createThemeModeVariants } from '@ses/core/utils/storybook/factories';
+import { figmaComparatorCommonPaddingOptions } from '@ses/core/utils/storybook/utils';
+import type { ComponentMeta } from '@storybook/react';
+export default {
+  title: 'Components/General/CheckBox',
+  component: CheckBox,
+} as ComponentMeta<typeof CheckBox>;
+
+const variantsArgs = [
+  {
+    isChecked: true,
+    label: 'Show empty budget categories',
+  },
+  {
+    isChecked: false,
+    label: 'Show empty budget categories',
+  },
+];
+
+export const [[Checked, Unchecked]] = createThemeModeVariants(CheckBox, variantsArgs);
```

Comment:
Please, note that `createThemeModeVariants` return 2 stories for each item in `variantsArgs`. The expected return value for your case would be:

```js
export const [[Checked, CheckedDarkMode], [Unchecked, UncheckedDarkMode]] = createThemeModeVariants(CheckBox, variantsArgs);
```